src/ripple/app/misc/SHAMapStoreImp.cpp

```diff
@@ -541,7 +548,42 @@ SHAMapStoreImp::clearSql (DatabaseCon& database,
             std::this_thread::sleep_for (
                     std::chrono::milliseconds (setup_.backOff));
     }
-    journal_.debug << "finished: " << deleteQuery;
+    JLOG(journal_.debug) << "finished: " << deleteQuery;
+}
+
+void
+SHAMapStoreImp::clearSql(DatabaseCon& database,
```

Comment:
I'm reluctant to inline that much code, even though it's only called once, particularly since `clearPrior` is cleaning up 3 other tables.